evap/static/js/sortable_query_binding.js

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
so much code. i especially like the `/* CODE */`. that's helpful. but you're the clean-up guy. so if you're okay with it, it's fine.